config/default.json

```diff
@@ -143,6 +143,7 @@
     },
     "centrifugo": {
         "apiUrl": "http://127.0.0.1:8001/api",
+        "publicUrl": "ws://localhost:8001/connection/websocket",
```

Comment:
i'd rather have the path omitted in this case, that would allow using another connection method (or allowing multiple as fallbacks) if necessary later on.

Reply:
resolved in https://github.com/Corsace/Corsace/pull/275/commits/23e0e1655eff917c68d6441dfea17c25042e59c4